src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents.mdx

```diff
@@ -99,22 +107,24 @@ Each issue preference option has its own advantages. You can select any preferen
 
       <td>
         An issue will be created for every incident in your policy.
+      </td>
 
+      <td>
         * The most notifications
-        * Useful if you need to be notified of every incident or if you have an external system where you want to send alert notifications
+        * Useful if you need to be notified of every incident in an entity or if you have an external system where you want to send alert notifications
```

Comment:
Incidents don't occur "in" an entity, they occur "for" an entity, or "about" an entity, or maybe "involving" an entity.

I would suggest "if you need to be notified about any entity that is in violation."

Reply:
I've replaced it with this: Useful if you need a notification about any entity that creates an incident or if you have an external system where you want to send alert notifications.
We avoid using "violation".